### DISCUSSION TITLE
Unable to create a new file in Remix. Lesson 1

### DISCUSSION THREAD

**Discussion by iamharman** [smartcontractkit/full-blockchain-solidity-course-py#664] [Q&A] [answered]
I am not able to create a new Solidity file in Remix. I'm using Safari, is this a browser issue.
I am not able to enter the name of this file so can't use it.

<img width="1440" alt="Screenshot 2022-01-01 at 12 03 22 AM" src="https://user-images.githubusercontent.com/37957496/147835630-d9ebc180-b7f2-4b09-82ea-23cb5ac7efd3.png">
?

**Comment by cromewar** [comment, marked as answer]
Is not recommended to use safari for this, you could try deleting the browser cache and try again, but is highly recommended to use something like chrome or brave.

**Reply by iamharman** [reply]
Thanks,
I switched to Brave and its working fine there.
